routes/articles-router.js

```diff
@@ -0,0 +1,11 @@
+const articlesRouter = require('express').Router();
+const {getArticleById, getArticles, patchArticleById} = require('../controllers/article-controllers')
+const {getArticleComments, postComment} = require('../controllers/comments-controllers')
+
+articlesRouter.get('/', getArticles)
+articlesRouter.get('/:article_id', getArticleById)
+articlesRouter.patch('/:article_id', patchArticleById)
```

Comment:
Looking great! If you want, when the path is the same, e.g. /:aricle_id, you can chain the methods. So you could have....

articlesRouter
  .route("/:article_id")
  .get(getArticleById)
  .patch(patchArticleById)
  
  Don't have to! Whatever you prefer

Reply:
that is so clean I love it!